src/figuresClasses.ts

```diff
@@ -1,19 +1,72 @@
-export interface Figure {
+const hackerAlert = new Error('Congratz! You\'ve hacked the system!');
 
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle',
+  color: 'red' | 'green' | 'blue',
+  getArea: () => number,
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Figure['shape'] = 'triangle';
```

Comment:
and then this would omit using strings - which are more prone to mistyping compared to enums
```suggestion
  shape: Figure.shape = Shape.triangle;
```